### PR TITLE
[1.16.5] Added full Spanish (es_es) language support [FINISHED]

### PR DESCRIPTION
**Pull request by FrannDzs**
Hello! How are you guys? I hope it's ok.
After 2 years without translation updates, and hard work on my part, I redid the complete Spanish translation from scratch.
(This may contain some context errors, which will be fixed quickly)
Regards!

### CODE REVIEW

File: src/main/resources/assets/immersiveengineering/manual/es_es/es_es.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (ravenbuilder934) [inline src/main/resources/assets/immersiveengineering/manual/es_es/es_es.md]: why was this added?

Comment (FrannDzs) [reply src/main/resources/assets/immersiveengineering/manual/es_es/es_es.md]: > why was this added?
I will translate the engineer manual completely in Spanish soon, and I put that just to remember it, sorry if it bothers there

Comment (FrannDzs) [reply src/main/resources/assets/immersiveengineering/manual/es_es/es_es.md]: > why was this added?

Tomorrow I will finish everything and update the PR

**Comment by FrannDzs** [comment]
A bunch of fixes. Corrected Engineer's Manual.
I would believe that there are no missing fixes. (unsuccessful) 99.9% complete

File: src/main/resources/assets/immersiveengineering/manual/es_es/es_es.md (hunk 1 above)
Comment (FrannDzs) [reply src/main/resources/assets/immersiveengineering/manual/es_es/es_es.md]: 0.1% to complete. I worked hard on it today. In a few hours I'll give it a final review and let you know when it's finally ready. I just hope they don't update before so I don't get the translation left out :( Let me know if they are about to release it please. Regards!

Comment (FrannDzs) [reply src/main/resources/assets/immersiveengineering/manual/es_es/es_es.md]: > why was this added?

Hi! Wow, this was a colossal job, it took me much longer than I thought it would. I have done a final check of all the files and strings and fixed a lot of inconsistencies everywhere. I'm done with the translation. I like to do things right and maybe I'm too detail oriented, but in the end it's a quality job.
Whenever you want you can merge the changes.
Thanks for your patience and the truth is that the pull request is full of commits, I don't know if it's the cleanest way to do it but let me know how I can improve. Thanks and greetings! Nothing more to say.

**Comment by BluSunrize** [comment]
I wish you had waited with this PR until you were actually finished, because good lord, my inbox.

**Comment by FrannDzs** [comment]
> I wish you had waited with this PR until you were actually finished, because good lord, my inbox.

Hello! I'm very sorry, because I was in a hurry I thought it was over and it has happened to me 10 times. I know it sounds weird but IE is not easy to translate added to the fact that I do not have much experience in GitHub and by the time I discovered the draft it was late lol. For next time I will keep it in mind. Greetings!!

**Comment by FrannDzs** [comment]
@BluSunrize After a very thorough examination, this is as I wanted it to be. I have checked every word, every article and corrected some spelling problems and the color codes and links.
You will rest from me for a while xD
Thanks guys, keep it up, cheers!
PD: If any Spanish-speaking person finds an error, do not hesitate to fix it or report it.

**Comment by BluSunrize** [comment]
Thank you =)
